src/screens/Game/GameSubMenu/index.tsx

```diff
@@ -191,6 +195,12 @@ export default function GamesSubmenu({
                 {t('submenu.addShortcut', 'Add shortcut')}
               </button>
             )}
+            <button
```

Comment:
can we detect if steam is actually installed? since we are looking for a steam folder, maybe we can hide this button if not

Reply:
We could only on mac and linux i think. With `which steam` we can atleast check that the executable is there.